classes/publication/Publication.inc.php

```diff
@@ -36,6 +36,30 @@ public function getEditorString() {
 		return join(__('common.commaListSeparator') . ' ', $editorNames);
 	}
 
+	/**
+	 * Get the URL to a cover image
+	 *
+	 * @param int $contextId
+	 * @param string $preferredLocale Return the cover image in a specified locale.
+	 * @return string
+	 */
+	public function getCoverImageUrl($contextId, $preferredLocale = null) {
```

Comment:
This function seems to do the same as the existing `getLocalizedCoverImageUrl` -- so maybe to use the existing one?